README.md

```diff
@@ -13,7 +13,7 @@ service: Microsoft Graph
 
 ![Build Badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/523/badge)
 
-| [Library](https://github.com/AzureAD/microsoft-authentication-library-for-objc) | [API Reference](https://azuread.github.io/docs/objc/) | [Support](README.md#community-help-and-support)
+| [Library](https://github.com/AzureAD/microsoft-authentication-library-for-objc) | [API Reference](https://azuread.github.io/docs/objc/) | [Support](README.md#feedback,-community-help,-and-support)

```

Comment:
Seems the link of Support does not work for me. It works if I changed the url to `README.md#feedback-community-help-and-support`

Reply:
Fixed. Thanks  :)